crates/node-types/src/http_server.rs

```diff
@@ -0,0 +1,144 @@
+use attestation::attestation::Attestation;
+use ed25519_dalek::VerifyingKey;
+use serde::{Deserialize, Serialize};
+
+pub mod verifying_key_bs58;
+
+#[derive(Clone, Serialize, Deserialize)]
+pub struct StaticWebData {
+    #[serde(with = "verifying_key_bs58")]
+    pub near_signer_public_key: VerifyingKey,
+
+    #[serde(with = "verifying_key_bs58")]
+    pub near_p2p_public_key: VerifyingKey,
+
+    #[serde(with = "verifying_key_bs58::vec")]
+    pub near_responder_public_keys: Vec<VerifyingKey>,
+
+    pub tee_participant_info: Option<Attestation>,
+}
+
+#[cfg(test)]
+#[allow(non_snake_case)]
+mod tests {
+    use super::*;
+    use ed25519_dalek::{SigningKey, VerifyingKey};
+    use rand_chacha::{ChaCha20Rng, rand_core::SeedableRng};
+
+    fn gen_verifying_key(seed: u64) -> VerifyingKey {
+        let mut rng = ChaCha20Rng::seed_from_u64(seed);
+        let sk = SigningKey::generate(&mut rng);
+        sk.verifying_key()
+    }
+
+    #[test]
+    fn verifying_key_bs58__should_serialize_to_exact_snapshot() {
```

Comment:
Is the double underscore intentional? Why not replace `__` with a single `_`? (The same question applies to several other places in the code.)